PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+## Overview
+<!--
+READ ME FIRST:
+Please answer *all* questions below and check off every point from the Essential Checklist!
+If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
+-->
+
+1. This PR fixes or fixes part of #[fill_in_number_here][(fill_in_the_issue_link)].
+2. This PR does the following: [Explain here what your PR does and why]
+3. (For bug-fixing PRs only) The original bug occurred because: [Explain what
+   the cause of the bug was, and which PR introduced it]
+
+## Essential Checklist
+
+- [ ] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...", followed by a short, clear summary of the changes.
+- [ ] I have followed the build process and error check mentioned in the [readme file](https://github.com/oppia/guppy/blob/master/README.md).
+- [ ] I have followed the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
+- [ ] I have assigned the correct reviewers to this PR (or will leave a comment with the phrase "@{{reviewer_username}} PTAL" if I don't have permissions to assign reviewers directly).
+- [ ] The linter/Karma presubmit checks pass on my local machine, and my PR follows the [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide)).
+- [ ] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
+
+
+## Proof that changes are correct
+
+<!--
+Add videos/screenshots of the user-facing interface to demonstrate that the changes
```

Comment:
Can you update these instructions to be more specific to Guppy? Remove any that don't apply, and explain how to show proof of changes.

Reply:
Done!

Reply:
@seanlip I have already the PR template specific to guppy.